types/i18next-express-middleware/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: http://i18next.com/
 // Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
```

Comment:
@DovydasNavickas Where are they used? As I see i18next-express-middleware and i18next-node-fs-backend tsconfig maps `i18next` to `i18next/v2` which does not use any TypeScript 2.1+ features and does not specify TypeScript version in its header.